scripts/hubot-netrunnerdb-cards.coffee

```diff
@@ -314,3 +343,27 @@ module.exports = (robot) ->
 				cardString += " + " + randomCard.title
 
 		res.send cardString
+
+	robot.hear /^!find (.*)/, (res) ->
+		conditions = []
+		for part in res.match[1].toLowerCase().split(" ")
```

Comment:
It looks like there may be an issue if you want to search for things like multi-word subtypes. On NRDB I can search for `s:"code gate"` to return Code Gates, but it looks like this code would split that into two separate search queries. :/